src/tests/multihost/alltests/test_services.py

```diff
@@ -200,3 +202,98 @@ def test_0006_bz1909755(self, multihost, backupsssdconf):
+        # port bash to pytest
+        tools = sssdTools(multihost.client[0])
+        sssd_params = {'domains': 'LDAP',
+                       'debug_level': '9'}
+        tools.sssd_conf('sssd', sssd_params)
+        domain_section = 'domain/LDAP'
+        domain_params = {'id_provider': 'ldap',
+                         'ldap_uri': 'ldap://ldap.example.com',
+                         'ldap_search_base': 'dc=example,dc=com',
+                         'auth_provider': 'krb5',
+                         'krb5_server': 'kerberos.example.com',
+                         'krb5_realm': 'EXAMPLE.COM',
+                         'debug_level': '9'}
+        tools.sssd_conf(domain_section, domain_params)
+        # stop sssd, delete logs and cache, start sssd
+        tools.clear_sssd_cache()
+        random_file1 = ''.join(random.choice(string.ascii_lowercase) for i in range(5))
+        take_bk = 'cp -f /etc/krb5.conf /etc/krb5.conf.backup'
+        multihost.client[0].run_command(take_bk, raiseonerr=False)
+        cmd_to_add = '{ echo "includedir /var/lib/sss/pubconf/krb5.include.d/"; cat /etc/krb5.conf; } > ' \
+                     f'/tmp/{random_file1}'
+        multihost.client[0].run_command(cmd_to_add, raiseonerr=False)
+        copy_radom = f'cp -f /tmp/{random_file1} /etc/krb5.conf'
+        multihost.client[0].run_command(copy_radom, raiseonerr=False)
+        start_sssd = multihost.client[0].service_sssd('start')
+        assert start_sssd == 0
+        restore_krb = 'cp -f /etc/krb5.conf.backup /etc/krb5.conf'
+        random_file2 = ''.join(random.choice(string.ascii_lowercase) for i in range(5))
+        multihost.client[0].run_command(restore_krb, raiseonerr=False)
+        cmd_with_sp = '{ echo "includedir /var/lib/sss/pubconf/krb5.include.d/    "; cat /etc/krb5.conf; } > ' \
+                      f'/tmp/{random_file2}'
+        multihost.client[0].run_command(cmd_with_sp, raiseonerr=False)
+        copy_radom2 = f'cp -f /tmp/{random_file2} /etc/krb5.conf'
+        multihost.client[0].run_command(copy_radom2, raiseonerr=False)
+        multihost.client[0].service_sssd('stop')
+        tools.remove_sss_cache('/var/log/sssd')
+        # Here sssd expected to fail, and if we use function from common to start, it will
+        # raise an exception, so added command to check log message after sssd fails to start.
+        start_sssd_sp = multihost.client[0].run_command('systemctl start sssd', raiseonerr=False)
+        assert start_sssd_sp.returncode != 0
+        log_str = multihost.client[0].get_file_contents('/var/log/sssd/sssd_LDAP.log').decode('utf-8')
+        assert re.compile(r'Failed to init Kerberos context .Included profile directory could not be read').search(
+            log_str)
+        multihost.client[0].run_command(restore_krb, raiseonerr=False)
+        multihost.client[0].run_command('rm -f /etc/krb5.conf.backup', raiseonerr=False)
+        multihost.client[0].run_command('rm -f /tmp/random*', raiseonerr=False)
```

Comment:
The files in /tmp are not created with `random` prefix, so they would not get deleted. The file filename should be eg.
`random_file1 = 'random' + ''.join(random.choice(string.ascii_lowercase) for i in range(5))`

Reply:
updated